tests/test_mqtt_wildcards.py

```diff
@@ -0,0 +1,20 @@
+import fnmatch
+import re
+from dmqtt.signals import convert_wildcards
+
+
+def test_single_level_wildcard():
+    pattern = convert_wildcards("home/+/status")
+    assert re.match(pattern, "home/kitchen/status")
+    assert not re.match(pattern, "home/kitchen/lights/status")
+
+def test_multi_level_wildcard():
+    pattern = convert_wildcards("home/#")
+    assert re.match(pattern, "home/kitchen/status")
+    assert re.match(pattern, "home/kitchen/lights/status")
+
+def test_mixed_wildcards():
+    pattern = convert_wildcards("home/+/lights/#")
+    assert re.match(pattern, "home/kitchen/lights/status")
+    assert re.match(pattern, "home/bathroom/lights/brightness/set")
+    assert not re.match(pattern, "home/status")
```

Comment:
```suggestion
from dmqtt.signals import convert_wildcards


from . import BaseTest


class WildcardTest(BaseTest):
    def test_single_level_wildcard(self):
        pattern = convert_wildcards("home/+/status")
        assert pattern.match("home/kitchen/status")
        assert not pattern.match("home/kitchen/lights/status")

    def test_multi_level_wildcard(self):
        pattern = convert_wildcards("home/#")
        assert pattern.match("home/kitchen/status")
        assert pattern.match("home/kitchen/lights/status")

    def test_mixed_wildcards(self):
        pattern = convert_wildcards("home/+/lights/#")
        assert pattern.match("home/kitchen/lights/status")
        assert pattern.match("home/bathroom/lights/brightness/set")
        assert not pattern.match("home/status")
```

Need to update this to a model based test. Since convert_wildcards returns a regex, we fix it here as well.